src/client/commands/autoalloc.rs

```diff
@@ -0,0 +1,204 @@
+use crate::client::globalsettings::GlobalSettings;
+use crate::common::timeutils::ArgDuration;
+use crate::rpc_call;
+use crate::server::autoalloc::{AllocationEvent, AllocationEventHolder};
+use crate::server::bootstrap::get_client_connection;
+use crate::transfer::connection::ClientConnection;
+use crate::transfer::messages::{
+    AddQueueParams, AddQueueRequest, AutoAllocRequest, AutoAllocResponse, FromClientMessage,
+    ToClientMessage,
+};
+use clap::Clap;
+use cli_table::{print_stdout, Cell, CellStruct, Color, Style, Table};
+
+#[derive(Clap)]
+#[clap(setting = clap::AppSettings::ColoredHelp)]
+#[clap(setting = clap::AppSettings::Hidden)] // TODO: remove once autoalloc is ready to be used
+pub struct AutoAllocOpts {
+    #[clap(subcommand)]
+    subcmd: AutoAllocCommand,
+}
+
+#[derive(Clap)]
+enum AutoAllocCommand {
+    /// Display information about autoalloc state
+    Info,
+    /// Display event log for a specified allocation queue
+    Log(LogOpts),
+    /// Add new allocation queue
+    Add(AddQueueOpts),
+}
+
+#[derive(Clap)]
+#[clap(setting = clap::AppSettings::ColoredHelp)]
+pub struct AddQueueOpts {
+    /// Name of the allocation queue
+    name: String,
+
+    #[clap(subcommand)]
+    subcmd: AddQueueCommand,
```

Comment:
Maybe added some kind of "--dry" flag that do not activates the autoallocator but shows the command that will be used for asking resources?